### DISCUSSION TITLE
Copy image to final output folder?

### DISCUSSION THREAD

**Discussion by lucasnantonio** [11ty/eleventy#1945] [Q&A] [answered]
I have this input structure:

- /input
  - post1.md
  - post2.md
  - /images
    - img1.jpg
    - img2.jpg
    - img3.jpg

In my markdown files, images are referenced as:

```
[Image 1](/images/img1.jpg)
[Image 2](/images/img2.jpg)
```

But when eleventy generates the output...

- /site
  - /post1/index.html
  - /post2/index.html
  - /images
    - img1.jpg
    - img2.jpg
    - img3.jpg

...It adds one level of depth to the relative path, breaking all of them. I really want to keep a simple structure in the input folder and avoid having to create a folder for each post and their images. 

My ideal output would either copy all used images to the post folder, or modify the output reference to match the new scheme.

Any tips?

**Comment by pdehaan** [comment, marked as answer]
I think I'm a bit confused here.
In your input folder you have a template, `./input/post2.md`, which references absolute images like `![](/images/img2.jpg)`. And your output folder writes the file as `/post2/index.html` and your images are deployed as `/images/img2.jpg`.
I wouldn't expect any images to break once deployed (assuming you just deploy the /site output folder) since they're absolute URLs. If they were relative URLs, I could see that it could be a potential issue though.

If you have a simple repo w/ test case I can take a closer look. Or if not I can probably create it from scratch.

**Reply by pdehaan** [reply]
https://github.com/pdehaan/11ty-1945

**Reply by lucasnantonio** [reply]
My mistake. I was mixing my relative and absolute paths. Your solution works, thanks!
